uqcs/templates/form.mako

```diff
@@ -282,7 +290,7 @@
, {"value": "Graduate Diploma of Environmental Management", "data": "postgrad"}, {"value": "Graduate Certificate of Community Development", "data": "postgrad"}, {"value": "Graduate Diploma of Business Research Methods", "data": "postgrad"}, {"value": "Masters of International Economics and Finance", "data": "postgrad"}, {"value": "Masters of Tourism, Hotel and Event Management", "data": "postgrad"}, {"value": "Graduate Certificate of Business Administration", "data": "postgrad"}, {"value": "Graduate Certificate of International Relations", "data": "postgrad"}, {"value": "Graduate Diploma of Urban and Regional Planning", "data": "postgrad"}, {"value": "Masters of Molecular Biology Research Extensive", "data": "postgrad"}, {"value": "Graduate Certificate of Environmental Management", "data": "postgrad"}, {"value": "Graduate Diploma of Governance and Public Policy", "data": "postgrad"}, {"value": "Masters of International Commercial Law/Commerce", "data": "postgrad"}, {"value": "Graduate Certificate of Business Research Methods", "data": "postgrad"}, {"value": "Graduate Diploma of Magnetic Resonance Technology", "data": "postgrad"}, {"value": "Graduate Certificate of Peace and Conflict Studies", "data": "postgrad"}, {"value": "Graduate Certificate of Arts in Japanese Translation", "data": "postgrad"}, {"value": "Graduate Certificate of Governance and Public Policy", "data": "postgrad"}, {"value": "Masters of International Relations/International Law", "data": "postgrad"}, {"value": "Graduate Certificate of Arts in Japanese Interpreting", "data": "postgrad"}, {"value": "Graduate Certificate of Magnetic Resonance Technology", "data": "postgrad"}, {"value": "Graduate Certificate of Entrepreneurship and Innovation", "data": "postgrad"}, {"value": "Graduate Certificate of Writing, Editing and Publishing", "data": "postgrad"}, {"value": "Masters of Arts in Chinese Translation and Interpreting", "data": "postgrad"}, {"value": "Masters of International Economics and Finance/Commerce", "data": "postgrad"}, {"value": "Graduate Certificate of Pharmaceutical Industry Practice", "data": "postgrad"}, {"value": "Masters of Arts in Japanese Interpreting and Translation", "data": "postgrad"}, {"value": "Graduate Certificate of Mediation and Conflict Resolution", "data": "postgrad"}, {"value": "Graduate Certificate of Tourism, Hotel and Event Management", "data": "postgrad"}, {"value": "Masters of Development Economics/Economics and Public Policy", "data": "postgrad"}, {"value": "Masters of International Relations/Peace and Conflict Studies", "data": "postgrad"}, {"value": "Graduate Certificate of Alcohol, Tobacco and Other Drug Studies", "data": "postgrad"}, {"value": "Graduate Diploma of Arts in Japanese Interpreting and Translation", "data": "postgrad"}, {"value": "Graduate Certificate of Arts in Chinese Translation and Interpreting", "data": "postgrad"}, {"value": "Graduate Certificate of Magnetic Resonance and Positron Emission Tomography", "data": "postgrad"}];
```

Comment:
Perhaps we're getting to the point where this needs to be a static file loaded in? @jdcaperon may have some advice on this front.

Reply:
This should be a file that is lazy loaded after the page render, would also be good to set an appropriate cache policy on the file as well so that we aren't forcing a load of it each time